code/modules/research/mechfab_designs.dm

```diff
@@ -1038,4 +1038,151 @@
+	name = "Armor Plate (Marshal)"
+	category = "Exosuit Internals"
+	id = "exo_int_armor_gygax"
+	req_tech = list(TECH_MATERIAL = 5, TECH_ENGINEERING = 4, TECH_COMBAT = 2)
+	materials = list(DEFAULT_WALL_MATERIAL = 40000, MAT_DIAMOND = 8000)
+	build_path = /obj/item/mecha_parts/component/armor/marshal
+
+/datum/design/item/mechfab/exointernal/darkgygax_armor
+	name = "Armor Plate (Blackops)"
+	category = "Exosuit Internals"
+	id = "exo_int_armor_dgygax"
+	req_tech = list(TECH_MATERIAL = 5, TECH_ENGINEERING = 5, TECH_COMBAT = 4, TECH_ILLEGAL = 2)
+	materials = list(MAT_PLASTEEL = 20000, MAT_DIAMOND = 10000, MAT_GRAPHITE = 20000)
+	build_path = /obj/item/mecha_parts/component/armor/marshal/reinforced
+
+/datum/design/item/mechfab/exointernal/durand_armour
+	name = "Armor Plate (Military)"
+	id = "exo_int_armor_durand"
+	req_tech = list(TECH_MATERIAL = 5, TECH_ENGINEERING = 4, TECH_COMBAT = 2)
+	materials = list(DEFAULT_WALL_MATERIAL = 40000, MAT_PLASTEEL = 9525, "uranium" = 8000)
+	build_path = /obj/item/mecha_parts/component/armor/military
+
+/datum/design/item/mechfab/exointernal/marauder_armour
+	name = "Armor Plate (Cutting Edge)"
+	id = "exo_int_armor_marauder"
+	req_tech = list(TECH_MATERIAL = 8, TECH_ENGINEERING = 7, TECH_COMBAT = 6, TECH_ILLEGAL = 4)
+	materials = list(MAT_DURASTEEL = 40000, MAT_GRAPHITE = 8000, MAT_OSMIUM = 8000)
+	build_path = /obj/item/mecha_parts/component/armor/military/marauder
+
+/datum/design/item/mechfab/exointernal/phazon_armour
+	name = "Armor Plate (Janus)"
+	id = "exo_int_armor_phazon"
+	req_tech = list(TECH_MATERIAL = 6, TECH_ENGINEERING = 6, TECH_COMBAT = 6, TECH_ILLEGAL = 4)
+	materials = list(MAT_MORPHIUM = 40000, MAT_DURASTEEL = 8000, MAT_OSMIUM = 8000)
+	build_path = /obj/item/mecha_parts/component/armor/alien
+
+/datum/design/item/mechfab/exointernal/stan_hull
+	name = "Hull (Standard)"
+	category = "Exosuit Internals"
+	id = "exo_int_hull_standard"
+	req_tech = list(TECH_MATERIAL = 2, TECH_ENGINEERING = 2)
+	materials = list(DEFAULT_WALL_MATERIAL = 10000)
+	build_path = /obj/item/mecha_parts/component/hull
+
+/datum/design/item/mechfab/exointernal/durable_hull
+	name = "Hull (Durable)"
+	category = "Exosuit Internals"
+	id = "exo_int_hull_durable"
+	req_tech = list(TECH_MATERIAL = 2, TECH_ENGINEERING = 2)
+	materials = list(DEFAULT_WALL_MATERIAL = 8000, MAT_PLASTEEL = 5000)
+	build_path = /obj/item/mecha_parts/component/hull/durable
+
+/datum/design/item/mechfab/exointernal/light_hull
+	name = "Hull (Lightweight)"
+	category = "Exosuit Internals"
+	id = "exo_int_hull_light"
+	req_tech = list(TECH_MATERIAL = 3, TECH_ENGINEERING = 4)
+	materials = list(DEFAULT_WALL_MATERIAL = 5000, MAT_PLASTIC = 3000)
+	build_path = /obj/item/mecha_parts/component/hull/lightweight
+
+/datum/design/item/mechfab/exointernal/stan_gas
+	name = "Life-Support (Standard)"
+	category = "Exosuit Internals"
+	id = "exo_int_lifesup_standard"
+	req_tech = list(TECH_MATERIAL = 2, TECH_ENGINEERING = 2)
```

Comment:
So now robotics _requires_ research to make _any_ mechs whatsoever?

Reply:
Negatory, exosuits start with their 'default' components by.. default. Research is needed to produce -extra- components.